STATUS: INLINE FAIL SETUPTEARDOWN
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerEJBTest.testFindAndSetAttributesSetEJB
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerEJBTest.java
CLEANER: de.strullerbaumann.visualee.examiner.ExaminerTest.testCleanPrimitives
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/examiner/ExaminerTest.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.ClassFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/ClassFilterTest.java
ORIGINAL CLEANER SIZE: 45
NEW CLEANER SIZE: 1
ELAPSED TIME: 50.89199995994568
==========================
@@ -79,0 +79,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
